packages/components/src/components/hds/stepper/nav/panel.hbs

```diff
@@ -0,0 +1,17 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+<section
+  class={{this.classNames}}
+  ...attributes
+  role="tabpanel"
```

Comment:
The role shouldnt be tabpanel if it is non-interactive. The reason is setting an ARIA role sets expectations for the user on how the UI will work.